RuntimeUnityEditor.Core/Windows/ObjectTree/RootGameObjectSearcher.cs

```diff
@@ -74,15 +83,28 @@ public static IEnumerable<GameObject> FindAllRootGameObjects()
         /// </summary>
         public IEnumerable<GameObject> GetRootObjects()
         {
-            if (SceneIndexFilter >= 0)
+            RuntimeUnityEditorCore.Logger.Log(LogLevel.Debug, $"GetRootObjects called, CurrentSearchType={CurrentSearchType}, SceneIndexFilter={SceneIndexFilter}");
```

Comment:
This is a bit spammy, ideally it would at least measure time with StopWatch (or just be commented out).